gslib/utils/encryption_helper.py

```diff
@@ -34,7 +34,8 @@
     lambda: re.compile('projects/([^/]+)/'
                        'locations/([a-zA-Z0-9_-]{1,63})/'
                        'keyRings/([a-zA-Z0-9_-]{1,63})/'
-                       'cryptoKeys/([a-zA-Z0-9_-]{1,63})$'))
+                       'cryptoKeys/([a-zA-Z0-9_-]{1,63})/'
+                       'cryptoKeyVersions/([0-9]{1,63})$'))
```

Comment:
Looks like the ValidateCMEK function seems to be getting used at multiple places and not every time we pass a key with CryptoKeyVersions (e.g. if you are reading it from boto config)?. I have a feeling that this might break other workflows.

Reply:
You're right, it will break a lot of things that rely on the `CryptoKeyWrapper` class.